src/components/repoCard.js

```diff
@@ -51,20 +52,46 @@ const RepoCard = ({ repo }) => {
         <Text><span role="img" aria-label="star" style={{ verticalAlign: 'middle' }}>
           <StarIcon />
         </span> {repo.stargazers_count}</Text>
-        <Text><span role="img" aria-label="issues" style={{ verticalAlign: 'middle' }}>
+        <Text><span class="tooltip" role="img" aria-label="issues" style={{ verticalAlign: 'middle' }}>
           <IssueIcon />
+          <span class="tooltiptext">Open Issues</span>
         </span> {repo.open_issues_count}</Text>
         <a href={repo.html_url} style={{ color: 'currentColor' }}>
           <Text><span role="img" aria-label="github" style={{ verticalAlign: 'middle' }}>
             <GithubIcon />
           </span> Github</Text>
         </a>
-        <Text><span role="img" aria-label="code" style={{ verticalAlign: 'middle' }}>
-          <CodeIcon />
-        </span> {repo.language}</Text>
+        <Text>
+          <span class="tooltip" role="img" aria-label="language" style={{ verticalAlign: 'middle' }}>
+          {displayLogo()}{checkNullLang()}            
+          </span>
+        </Text>
       </Grid>
     </Card>
   )
-}
+  function displayLogo()
+  { 
+    if(languageLogoList.includes(repo.language))
+    {
+      var langLogo='https://raw.githubusercontent.com/aj-ya/github-covid-finder/pr/add_logos/src/images/icons/languages/'+repo.language.replace(/\s+/g,"_")+'.svg?sanitize=true';//loading the image form github and not local files.replace function to replace one or more whitespaces with a single underscore
```

Comment:
why not a simple mapper?
(just pseudo code below)
```
const logoMapper = {
   javascript: 'path/to/js/logo',
   ...
```
or

```
const logoMapper = {
   javascript: <img width="16px" height="16px" src={jsLogo} alt="js" />,
   ...
```

🤔 

Thanks for the help, if you want to try these suggestions feel free, I'll try to take a better look on Monday, as is cannot approve but a great contribution

Reply:
Thanks for the suggestion, will work on this today.